src/components/Theme/README.md

```diff
@@ -367,13 +369,144 @@ export default {
 </style>
 ```
 
+## Customizing the theme within subsets of the app
+
+Theme components can be nested, and when nested the child Theme can override its parent Theme in two ways:
+1. Providing its own theme data, which will be merged with the theme data provided by the parent or
+2. Selecting a profile from the parent theme, which will be merged with the theme data provided by the parent
+
```

Comment:
We'll need to think about how to standardize the usage. I can imagine devs adding their own profiles in subsets of the app, which it looks like we are discouraging.